src/fides/api/ops/service/privacy_request/request_runner_service.py

```diff
@@ -679,28 +659,40 @@ def _retrieve_child_results(  # pylint: disable=R0911
     return results
 
 
-def needs_consent_email_send(
-    db: Session, user_identity: Dict[str, Any], privacy_request: PrivacyRequest
+def get_consent_email_connection_configs(db: Session) -> Query:
+    """Return enabled consent email connection configs."""
+    return db.query(ConnectionConfig).filter(
+        ConnectionConfig.connection_type.in_(CONSENT_EMAIL_CONNECTOR_TYPES),
+        ConnectionConfig.disabled.is_(False),
+        ConnectionConfig.access == AccessLevel.write,
+    )
+
+
+def get_erasure_email_connection_configs(db: Session) -> Query:
+    """Return enabled erasure email connection configs."""
+    return db.query(ConnectionConfig).filter(
+        ConnectionConfig.connection_type.in_(ERASURE_EMAIL_CONNECTOR_TYPES),
+        ConnectionConfig.disabled.is_(False),
+        ConnectionConfig.access == AccessLevel.write,
+    )
+
+
+def needs_email_send(
```

Comment:
Can we give this method a more specific name to distinguish from the privacy request completion notifications we make?

Reply:
Would `needs_batch_email_send` work, or is that still too generic? I wanted to avoid putting `needs_consent_or_erasure_email_send`

Reply:
yes that sounds good sorry missed this question earlier